.github/workflows/build_and_test.yaml

```diff
@@ -52,16 +50,33 @@ jobs:
         files: ./coverage.xml
         name: codecov-envoy-gateway
         verbose: true
+  
+  conformance-tests:
+    runs-on: ubuntu-latest
+    needs: [unit-tests]
+    strategy:
+      matrix:
+        kind_node_tag: ["v1.24.0", "v1.23.6", "v1.22.9"]
```

Comment:
this will increase PR time by 6min, I think thats fine for now

Reply:
They run in parallel though

Reply:
ah amazing !